extensions/azurecore/src/account-provider/azureAccountProvider.ts

```diff
@@ -50,6 +50,7 @@ export class AzureAccountProvider implements azdata.AccountProvider, vscode.Disp
 
 	clearTokenCache(): Thenable<void> {
 		return this.getAuthMethod().deleteAllCache();
+		return undefined;
```

Comment:
This isn't doing anything?

Reply:
Whoops i hooked these functions up later, forgot to remove the return undefined.